tsconfig.json

```diff
@@ -1,4 +1,5 @@
 {
+  "compileOnSave": true,
```

Comment:
Not sure if this impacts others? Can remove. As I'm using atom not vscode it was a bit annoying to have to `tsc` every time.

Reply:
its fine if it is true.